index.html

```diff
@@ -4,49 +4,49 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" type="text/css" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css" integrity="sha512-HK5fgLBL+xu6dm/Ii3z4xhlSUyZgTT9tuc/hSrtw6uzJOvgRr2a9jyxxT1ely+B+xFAmJKVSTbpM/CuL7qxO8w==" crossorigin="anonymous" />
```

Comment:
Don't use font awesome, I had given you a link for icons http://flaticon.com/
download icons from there and use them, you have to use images as icons

Reply:
how to use that ?

Reply:
have you opened the link? you have to search the icon that you want and download it

Reply:
yes , do i have to use the href tag for this?

Reply:
no, you have to download the image and use the image tag for this